lib/databases/mysql

```diff
@@ -161,7 +164,9 @@ EOF
     elif is_ubuntu; then
         install_package mysql-server
     else
-        exit_distro_not_supported "mysql installation"
+	install_package mariadb-server
```

Comment:
again, add the check using is_clearlinux to do this and leave the else clause in place as-is